Userland/Libraries/LibWeb/SRI/SRI.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023, Sam Atkins <atkinssj@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+
+namespace Web::SRI {
+
+// https://w3c.github.io/webappsec-subresource-integrity/#integrity-metadata
+struct Metadata {
+    String algorithm;    // "alg"
+    String base64_value; // "val"
+    String options {};   // "opt"
+};
+
+// https://w3c.github.io/webappsec-subresource-integrity/#apply-algorithm-to-response
```

Comment:
No spec links on declarations please (and then you can group them neatly together and get rid of the newlines)